### ISSUE TITLE
carousel_hestiaUI Will Look Better With Centering Text

### ISSUE THREAD

**Issue by CoryGalyna** [ZORALab/Hestia#62]
# Description
```
Please provide a short description of what you have encountered below.
```
On https://hestia.zoralab.com/en/docs/hestiaui/carousel_hestiaui/#design, the carousel's slides' text are left-aligned. Centering them will make the design cleaner and nice.

# Link
```
Please provide the documentation link(s) below.
```
https://hestia.zoralab.com/en/docs/hestiaui/carousel_hestiaui/#design

# Expected Presentation
```
Please specify the expected presentation of the documentation.
```
Centering text.
![screenshot-2023-02-06-05-00-03](https://user-images.githubusercontent.com/124406765/216845510-4f1df7de-2220-437e-8202-9359ca57041b.jpg)

# Current Presentation
```
Please specify the current presentation of the documentation.
```
Left-aligned text.

![screenshot-2023-02-06-04-59-20](https://user-images.githubusercontent.com/124406765/216845521-5e4bc983-3ea6-44a9-bea1-7b3bf8951eb7.jpg)

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Accepted.

**Comment by hollowaykeanho** [comment]
Fixed in 51b88730866fde221a2f2174357b8a399825d3ae

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
